lib/src/main/java/co/nimblehq/alpine/lib/model/nfc/AdditionalPersonDetails.kt

```diff
@@ -0,0 +1,19 @@
+package co.nimblehq.alpine.lib.model.nfc
+
+data class AdditionalPersonDetails(
+    var custodyInformation: String? = null,
+    var fullDateOfBirth: String? = null,
+    var nameOfHolder: String? = null,
+    var otherNames: List<String>? = null,
+    var otherValidTDNumbers: List<String>? = null,
+    var permanentAddressConstituents: List<String>? = null,
+    var personalNumber: String? = null,
+    var personalSummary: String? = null,
+    var placeOfBirthConstituents: List<String>? = null,
+    var profession: String? = null,
+    var proofOfCitizenship: ByteArray? = null,
+    var tag: Int = 0,
+    var tagPresenceList: List<Int>? = null,
```

Comment:
Can it be `tagPresences` to be consistent with another list name?

Reply:
@lydiasama it's not a list of `TagPresence` objects. From the documentation of the `DG11File.java` method, it returns a list of integers representing a list of tags actually present. 
![image](https://user-images.githubusercontent.com/8093908/179664791-db6e85af-f240-4030-ad0a-b5bd2f89d655.png)

So we can do three things:
* Keeping the name unchanged - `tagPresenceList` - to make it consistent with the method of the `DG11File`.
* Rename it to `tags` (not descriptive).
* Rename it to `presentTags` (sounds weird tho 😅 )

More suggestions are welcome. Your call.

Reply:
@AVI5HEK `presenceTags`?

Reply:
@AVI5HEK It's weird with all approaches tho. 😅 Even we keep it the same as `tagPresenceList` because we change the rest list to be a plural noun.

Reply:
We need help from Professor @ryan-conway .

Reply:
`tagPrecenseList` implies that the tags that are on the list are present, right? 
If yes, then I think `tags` is descriptive enough for this field since the tags are technically just `int`s. If the tag is in the list then we can consider it as "present" 💪🏻